src/main/java/core/basesyntax/mainoperation/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.mainoperation;
+
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.model.dto.impl.ReadFromFileImpl;
+import core.basesyntax.model.dto.impl.ReportHandlerImpl;
+import core.basesyntax.model.dto.impl.WriteToFileImpl;
+import core.basesyntax.service.fileservice.impl.FileParserImpl;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        List<String> linesFromFile = new ReadFromFileImpl().readFile("Operations.csv");
```

Comment:
```suggestion
        List<String> linesFromFile = new FileReaderImpl().readFile("Operations.csv");
```
Class names should be nouns